packages/angular-cli/lib/config/schema.json

```diff
@@ -135,10 +135,13 @@
         },
         "prefixInterfaces": {
           "type": "boolean"
+        },
+        "poll": {
+          "type": "number"
```

Comment:
@hansl ultimately it ends up being used here: https://github.com/webpack/watchpack/blob/master/lib/DirectoryWatcher.js#L58